staging/src/k8s.io/apiserver/pkg/server/options/encryptionconfig/types.go

```diff
@@ -77,10 +79,10 @@ type IdentityConfig struct{}
 // KMSConfig contains the name, cache size and path to configuration file for a KMS based envelope transformer.
 type KMSConfig struct {
 	// name is the name of the KMS plugin to be used.
-	Name string `json:"name"`
+	Name string
 	// cacheSize is the maximum number of secrets which are cached in memory. The default value is 1000.
 	// +optional
-	CacheSize int `json:"cachesize,omitempty"`
```

Comment:
@liggitt This type looks suspicious. Did we decide some time ago to always use integers with a fixed size in APIs, do we?

Reply:
the field in the external type should be a fixed size int, yes